### ISSUE TITLE
batchsize问题

### ISSUE THREAD

**Issue by maixiaolan** [CrossmodalGroup/LAPS#14]
CUDA_VISIBLE_DEVICES=0,1 python -m torch.distributed.run --nproc_per_node=2 train.py --dataset f30k --multi_gpu 1 --logger_name runs/f30k_vit --batch_size 64 --vit_type vit --embed_size 512 --sparse_ratio 0.5 --aggr_ratio 0.4这个命令可以模拟出单卡batch_size=128的表现吧？

**Comment by darkpromise98** [comment]
Theoretically, by leveraging PyTorch's _data parallelism_ to execute on multiple GPUs, it is possible to mimic the effect of running on a single GPU. 

Nevertheless, owing to the implementation of underlying operations like **data gather/scatter**, **tensor broadcast/reduce**, and gradient propagation, there will be subtle disparities in the actual outcomes.
